PWGDQ/Core/CutsLibrary.cxx

```diff
@@ -2043,13 +2050,29 @@ AnalysisCut* o2::aod::dqcuts::GetAnalysisCut(const char* cutName)
     return cut;
   }
 
-  if (!nameStr.compare("kaonPIDnsigma")) {
-    cut->AddCut(VarManager::kTPCnSigmaKa, -3.0, 3.0);
+  if (!nameStr.compare("kaonPIDnsigma2")) {
```

Comment:
You repeat kaonPIDnsigma2 3 times. Is this expected?

Reply:
Thank you for noticing! Should be corrected now

Reply:
I am very sorry. It seems you have removed kaonPIDnsigma. Is it on the purpose?